src/sentry/replays.py

```diff
@@ -1,10 +1,6 @@
 from __future__ import absolute_import
-
-import socket
-
-from httplib import HTTPConnection, HTTPSConnection
-from urllib import urlencode
-from urlparse import urlparse
+from requests import RequestException
```

Comment:
Isn't this `from requests.exceptions import RequestException`?

That's how we use it elsewhere.